readme.md

```diff
@@ -58,6 +58,14 @@ Create smart quotes (`boolean`, default: `true`).
 
 Converts straight double and single quotes to smart double or single quotes.
 
+###### `options.openingQuotes`
+
+Which characters to use for opening quotes `{ double: '”', single: '’' }`.
```

Comment:
```suggestion
Which characters to use for opening quotes `{double: '“', single: '‘'}`.
```

I believe other than whitespace I also fixed the quotes?

Reply:
oh you're right, my bad!